expression/builtin_info_vec_test.go

```diff
@@ -31,6 +32,9 @@ var vecBuiltinInfoCases = map[string][]vecExprBenchCase{
 	ast.CurrentRole:    {},
 	ast.TiDBIsDDLOwner: {},
 	ast.ConnectionID:   {},
+	ast.LastInsertId: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{}},
```

Comment:
Should be ```{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt}},```

Reply:
why? I think `LAST_INSERT_ID()` has no args/If add this it must be fuction `builtinLastInsertIDWithIDSig`.

Reply:
@Reminiscent PTAL